docs/reference/api/metadata-stores/delta.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Delta Lake Metadata Store
+---
+
+# DeltaMetadataStore
+
+::: metaxy.metadata_store.delta.DeltaMetadataStore
+
+## Installation
```

Comment:
Should come before the API reference OR be in `learn` docs instead

Reply:
done